ganga/GangaCore/Lib/Localhost/Localhost.py

```diff
@@ -231,7 +236,11 @@ def preparejob(self, jobconfig, master_input_sandbox):
         script = script.replace('###ENVIRONMENT###', repr(environment))
         script = script.replace('###WORKDIR###', repr(workdir))
         script = script.replace('###INPUT_DIR###', repr(job.getStringInputDir()))
-
+        script = script.replace('###VIRTUALIZATION###', repr(virtualization.__class__.__name__))
+        script = script.replace('###VIRTUALIZATIONIMAGE###', repr(virtualization.imageUrl))
+        script = script.replace('###VIRTUALIZATIONMODE###', repr(virtualization.mode) if hasattr(virtualization, 'mode') else repr(None)  )
+        print("app:")
+        print(repr(appscriptpath))
```

Comment:
Please remove prints from Ganga running code. If these are generally useful then please use the logging system otherwise remove them

Reply:
I removed them